### PR TITLE
chore: Removing unused incubator library

### PR DESCRIPTION
**Pull request by hackergil**
Incubator library won't be published anymore since we don't use it.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -349,43 +349,6 @@
         }
```
Comment (znikola) [inline angular.json:349]: Can you add some useful information to the PR description?
When merging to develop, this should automatically be copied by GH, and used in the release notes

File: angular.json [hunk 2]
```diff
@@ -349,43 +349,6 @@
...
       }
```
Comment (znikola) [inline angular.json:350]: We should add one line to our deprecation docs and mention about the removal of incubator lib

**Review by znikola — CHANGES_REQUESTED** [review]

File: angular.json (hunk 2 above)
Comment (hackergil) [reply angular.json:350]: Done

File: angular.json (hunk 1 above)
Comment (hackergil) [reply angular.json:349]: Done

**Review by znikola — APPROVED** [review]

**Pull request merged**